tsinfer/inference.py

```diff
@@ -947,8 +947,12 @@ def run(self):
             logger.info(f"Starting build for {self.num_ancestors} ancestors")
             progress = self.progress_monitor.get("ga_generate", self.num_ancestors)
             a = np.zeros(self.num_sites, dtype=np.int8)
-            root_time = max(self.timepoint_to_epoch.keys()) + 1
-            ultimate_ancestor_time = root_time + 1
+            root_time = max(self.timepoint_to_epoch.keys())
+            if root_time < 1:
+                root_time = 1  # The normal case, when time==freq
+            else:
+                root_time += 1  # Could happen with user-specified times
```

Comment:
I don't understand this comment - what could happen?

Reply:
We could have root_time >=1 (because the root time could be any number)